go/consensus/tendermint/roothash/roothash.go

```diff
@@ -435,6 +449,13 @@ func (sc *serviceClient) processFinalizedEvent(
 	reindex bool,
 ) (err error) {
 	tr := sc.trackedRuntime[runtimeID]
+	if tr == nil {
+		sc.logger.Error("runtime not tracked",
+			"runtime_id", runtimeID,
+			"trackedRuntimes", sc.trackedRuntime,
```

Comment:
```suggestion
			"tracked_runtimes", sc.trackedRuntime,
```
Also does including `sc.trackedRuntime` actually output a list or do you actually need to use something like `fmt.Sprintf("%+v", sc.trackedRuntime)`?

Reply:
seems like it does `"tracked_runtimes":{"gAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA=":{}`

Reply:
Cool!